Sources/Jobs/JobsConfiguration.swift

```diff
@@ -25,6 +29,21 @@ public struct JobsConfig {
         storage[key] = job
     }
     
+    /// Schedules a new job for execution at a later date.
+    /// Example: `config.schedule(job).daily().at(.startOfDay)`
+    /// - Parameter job: The `ScheduledJob` to be scheduled.
+    mutating public func schedule<J: ScheduledJob>(_ job: J) -> Scheduler {
+        let key = String(describing: J.self)
+        if let existing = scheduledStorage[key] {
+            print("WARNING: A scheduled job is already registered with key \(key): \(existing)")
```

Comment:
We should use apple/swift-log for stuff like this now

Reply:
Good idea, will update